tests/jballerina-unit-test/src/test/resources/test-src/expressions/let/let-expression-test.bal

```diff
@@ -316,6 +316,18 @@ function getRecordConstrainedError() returns FooError {
     return e;
 }
 
+public function testAnonymousRecordWithLetExpression() {
```

Comment:
Shall we add another test like

where `Rec` is
```ballerina
type Rec record {|
    int i;
    int j = 100;
|};
```

```ballerina
Rec rec1 = let int v = 160 in {i: v};
assert(160, rec1.i);
assert(100, rec1.j);

Rec rec2 = let int v = 161 in {i: v, j: v};
assert(161, rec2.i);
assert(161, rec2.j);
```

Reply:
Added!